src/MathOptInterface.jl

```diff
@@ -71,11 +72,16 @@ ConstraintMapping() = ConstraintMapping(
 
 mutable struct CplexSolverInstance <: MOI.AbstractSolverInstance
     inner::Model
+
     last_variable_reference::UInt64
     variable_mapping::Dict{MOI.VariableReference, Int}
+    variable_references::Vector{MOI.VariableReference}
```

Comment:
Why do we need `variable_mapping` and `variable_references`, cant we just use `keys(variable_mapping)`?